### ISSUE TITLE
python 3 support

### ISSUE THREAD

**Issue by txemi** [pynag/pynag#233]
Is seems it does not compile with python 3: 

```
⟫ sudo pip3 install pynag
[...]
Downloading from URL https://pypi.python.org/packages/source/p/pynag/pynag-0.9.1.tar.gz#md5=729eb229f0bb2288e73aca9448ff2a48 (from https://pypi.python.org/simple/pynag/)
  Running setup.py (path:/tmp/pip_build_root/pynag/setup.py) egg_info for package pynag
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip_build_root/pynag/setup.py", line 30
        print "Warning: Build of manpage failed \"%s\":\n%s\n%s" % (
                                                               ^
    SyntaxError: invalid syntax
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip_build_root/pynag/setup.py", line 30

    print "Warning: Build of manpage failed \"%s\":\n%s\n%s" % (

                                                           ^

SyntaxError: invalid syntax
[...]
```

**Comment by leonaard** [comment]
I made the code py3 compatible this way:

    git clone https://github.com/pynag/pynag python2-pynag
    2to3 --output-dir=python3-pynag -W -n python2-pynag
    rsync -r -u python2-pynag/* python3-pynag

and then installed system-wide:

    cd python3-pynag
    sudo python3 setup.py install

it seems to work, but i dont know if the output code is good enough to be pushed on `master` :)

**Comment by BobWhitelock** [comment]
I tried converting the code to support Python 3 as suggested by @leonaard, however I found that this gave many errors when I imported and tried to run it. Amongst other things it looks like there are various issues with byte/string handling which `pynag` relies on, which are changed between Python 2 and 3 and are unhandled by `2to3`. Once I got them to run, there was a lot of failing/erroring tests as well.

Has anyone managed to get this codebase working from Python 3 yet? Currently I think my best bet is having wrapper scripts around the Pynag behaviour I want, and running these from my Python 3 code. Thanks
